src/components/lab/ListItemView/ListItemView.scss

```diff
@@ -0,0 +1,285 @@
+@use '../../variables';
+@use '../../../../styles/mixins';
+
+$block: '.#{variables.$ns}lab-list-item-view';
+
+/* ListItemView CSS API */
+#{$block} {
+    /* Sizes */
+    --_--g-list-item-view-height: var(--g-list-item-view-height, #{variables.$m-height});
+    --_--g-list-item-view-border-radius: var(
+        --g-list-item-view-border-radius,
+        var(--g-border-radius-m)
+    );
+    --_--g-list-item-view-padding-inline: var(
+        --g-list-item-view-padding-inline,
+        var(--g-spacing-2)
+    );
+    --_--g-list-item-view-padding-block: var(--g-list-item-view-padding-block, var(--g-spacing-1));
+    --_--g-list-item-view-line-height: var(--g-list-item-view-line-height, 18px);
+
+    --_--g-list-item-view-controls-gap: var(--g-list-item-view-controls-gap, var(--g-spacing-1));
+    --_--g-list-item-view-controls-size: var(
+        --g-list-item-view-controls-size,
+        #{variables.$s-height}
+    );
+    --_--g-list-item-view-controls-border-radius: var(
+        --g-list-item-view-controls-border-radius,
+        var(--g-border-radius-s)
+    );
+    --_--g-list-item-view-controls-icon-size: var(--g-list-item-view-controls-icon-size, 16px);
+    --_--g-list-item-view-spacer-size: var(
+        --g-list-item-view-spacer-size,
+        var(--_--g-list-item-view-controls-size)
+    );
+
+    /* Colors */
+    --_--g-list-item-view-background: var(--g-list-item-view-background, var(--g-color-base-fill));
+    --_--g-list-item-view-background-hover: var(
+        --g-list-item-view-background-hover,
+        var(--g-color-base-simple-hover)
+    );
+    --_--g-list-item-view-background-disabled: var(
+        --g-list-item-view-background-disabled,
+        var(--g-color-base-fill)
+    );
+    --_--g-list-item-view-background-selected: var(
+        --g-list-item-view-background-selected,
+        var(--g-color-base-selection)
+    );
+    --_--g-list-item-view-background-selected-hover: var(
+        --g-list-item-view-background-selected-hover,
+        var(--g-color-base-selection-hover)
+    );
+    --_--g-list-item-view-focus-outline-width: var(--g-list-item-view-focus-outline-width, 2px);
+    --_--g-list-item-view-focus-outline-style: var(--g-list-item-view-focus-outline-style, solid);
+    --_--g-list-item-view-focus-outline-color: var(
+        --g-list-item-view-focus-outline-color,
+        var(--g-color-line-focus)
+    );
+    --_--g-list-item-view-focus-outline-offset: var(--g-list-item-view-focus-outline-offset, 0);
+    --_--g-list-item-view-color: var(--g-list-item-view-color, var(--g-color-text-primary));
+    --_--g-list-item-view-color-description: var(--g-color-text-secondary);
+    --_--g-list-item-view-color-disabled: var(
```

Comment:
We don't need this props, it overlaps with text-color, and description-color

Reply:
It's used, why it is not needed?